projects/ui/src/lib/components/po-table/po-table.component.spec.ts

```diff
@@ -2695,4 +2695,65 @@ describe('PoTableComponent:', () => {
 
     expect(component['hasInfiniteScroll']()).toBeFalse();
   });
+
+  it('remove: should remove item if received element index', () => {
```

Comment:
```suggestion
  it('remove: should remove the item by index', () => {
```
A tradução não representa o cenário